pb-cli.js

```diff
@@ -515,11 +515,12 @@ var updatedChatResp = function(input){
 	var jObj = JSON.parse(body);
 	if (jObj.status === 'ok') {
       if(jObj.client_name && jObj.client_name != nconf.get('client_name')) {
+        console.log('client_name:'+ jObj.client_name);
         nconf.set('client_name', jObj.client_name)
       }
 	    nconf.set('sessionid', jObj.sessionid);
 	    jObj.responses.forEach (function (entry) {
-	        console.log(conf_botname() + '> ' + entry);
+	        console.log('bot > ' + entry);
```

Comment:
We can do \`${conf_botname() || 'bot'}> ${entry}\` to default to bot only if there's no botname.